src/OSPSuite.R/Services/ImportConfigurationTask.cs

```diff
@@ -0,0 +1,115 @@
+﻿using OSPSuite.Core.Import;
+using OSPSuite.Infrastructure.Import.Core;
+using OSPSuite.Infrastructure.Import.Extensions;
+using System.Linq;
+using ImporterConfiguration = OSPSuite.Core.Import.ImporterConfiguration;
+
+namespace OSPSuite.R.Services
+{
+   public interface IImportConfigurationTask
+   {
+      MappingDataFormatParameter GetTime(ImporterConfiguration configuration, ColumnInfo[] columnInfos);
```

Comment:
Why all this Get and Set stuff? Looks very weird. 
Also GetTime returns a MappingDataFormatParameter? I don't understand
Can you also please add comments to the public method of. the interface so that we understand what's going on and what those parameters mean?